src/www/ui_tests/api/Controllers/GroupControllerTest.php

```diff
@@ -179,12 +329,8 @@ private function getUsersWithGroup($userIds)
    */
   public function testDeleteGroup()
   {
-    $groupName = 'fossy';
     $groupId = 4;
     $userId = 1;
-    $request = M::mock(Request::class);
-    $request->shouldReceive('getAttribute')->andReturn(ApiVersion::V1);
```

Comment:
Tests will fail without setting the api version.

Reply:
Resolved! Thank you @GMishx